website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,125 @@
+  </C.Property>
+  <C.Property @name="yield" @type="yield">
+    It's possible to yield generic content to the "toggle" element, instead of using the `@text` argument. This should be used only in special/edge cases where the toggle based on text/icon doesn't work for the specific context or design.
+    <br />
+    <br />
+    _⚠️ Important: in this case it's up to the consumers to make sure the implementation is compliant with the [accessibility requirements](/components/rich-tooltip?tab=accessibility)._
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+#### RichTooltip::Bubble
+
+Generic container that yields its content inside the "tooltip" popover element.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of the tooltip's bubble.
+    <br />
+    <br />
+    _⚠️ Important: we apply a CSS reset (`all: initial`) to the container, to avoid that styles applied to the parent elements leak into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support)._
+  </C.Property>
+  <C.Property @name="placement" @type="enum" @values={{array "top" "top-start" "top-end" "right" "right-start" "right-end" "bottom" "bottom-start" "bottom-end" "left" "left-start" "left-end"}} @default="bottom">
+    Placement for the preferred starting position of the tooltip's bubble relative to the toggle element.
+    <br />
+    <br />
+    _Notice: if `@enableCollisionDetection` is set, the bubble will automatically shift position to remain visible when near the edges of the screen regardless of the starting placement._
+  </C.Property>
+  <C.Property @name="offset" @type="number|object" @default="12">
+    A number that represents the “distance” between the tooltip's bubble and the toggle element (used to control the relative position of the arrow in relation to the toggle). An object can also be passed, which enables you to individually configure a different axis.
+    <br />
+    <br />
+    For details see: [Floating UI > Offset > Options](https://floating-ui.com/docs/offset#options).
+  </C.Property>
+  <C.Property @name="enableCollisionDetection" @type="boolean|string" @values={{array "true" "false" "flip" "shift" "auto"}} @default="false">
+    This property controls if the tooltip should automatically adapt its position to remain visible when near the edges of the viewport. It can be enabled for both axes by setting it to `true`, or a single axis can be chosen by passing the `flip` or `shift` value. If set to `auto` it will automatically place the popover in the position where there's more space available, but in this case it will ignore the `placement` value.
```

Comment:
```suggestion
    This property controls whether the tooltip should automatically adapt its position to remain visible when near the edges of the viewport. It can be enabled for both axes by setting it to `true` or for a single axis by passing either the `flip` or `shift` values. If set to `auto`, it will automatically place the popover in the position where there's more space available, but in this case, it will ignore the `placement` value.
```